hack/setup-guard-service.sh

```diff
@@ -0,0 +1,6 @@
+kubectl apply -f config/gateAccount.yaml
+kubectl apply -f config/serviceAccount.yaml
+kubectl apply -f config/guardiansCrd.yaml
```

Comment:
Maybe make a `config/resources` directory for these files and then setup can be just `kubectl apply -f config/resources` ?